assets/js/mbta.js

```diff
@@ -61,769 +61,4535 @@ const branchConfig = {
+            modes: {
+              auto: true, custom: true, headway: true, off: true,
+            },
+          },
+          e: {
+            value: false,
+            modes: {
+              auto: true, custom: true, headway: true, off: true,
+            },
+          },
+          w: {
+            value: false,
+            modes: {
+              auto: true, custom: true, headway: true, off: true,
+            },
+          },
+          c: {
+            value: false,
+            modes: {
+              auto: true, custom: true, headway: true, off: true,
+            },
+          },
+          m: {
+            value: false,
+            modes: {
+              auto: true, custom: true, headway: true, off: true,
+            },
+          },
+        },
       },
-    },
-  ],
-  SL3: [
-    {
-      id: 'SSOU',
-      name: 'South Station',
-      zones: {
-        n: false, s: false, e: true, w: false, c: false, m: false,
+    ],
+  },
+  SL3: {
+    batchModes: { auto: true, headway: false, off: true },
+    stations: [
+      {
+        id: 'SSOU',
+        name: 'South Station',
+        zones: {
+          n: {
+            value: false,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          s: {
+            value: false,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          e: {
+            value: true,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          w: {
+            value: false,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          c: {
+            value: false,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          m: {
+            value: false,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+        },
       },
-    },
-    {
-      id: 'SCOU',
-      name: 'Courthouse',
-      zones: {
-        n: false, s: false, e: true, w: true, c: false, m: true,
+      {
+        id: 'SCOU',
+        name: 'Courthouse',
+        zones: {
+          n: {
+            value: false,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          s: {
+            value: false,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          e: {
+            value: true,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
+            },
+          },
+          w: {
+            value: true,
+            modes: {
+              auto: false, custom: false, headway: false, off: true,
```

Comment:
I think this should have `auto` as an option (the Silver Line has good predictions in the South Station direction IIRC).

Reply:
This is from the ticket. Are you saying that all three of these stations should have only "off" on the eastbound sign, and all other signs should have "auto" too?
`the mode select box on South Station, Courthouse, and WTC EB signs only offers "off"`

Reply:
Yeah my interpretation was that Courthouse and WTC should allow `auto` in the WB direction (which I believe is already how you have it for WTC, incidentally?)